R/render.R

```diff
@@ -964,7 +964,7 @@ render <- function(input,
     # if the output format is LaTeX, first convert .md to .tex, and then convert
     # .tex to .pdf via latexmk() if PDF output is requested (in rmarkdown <=
     # v1.8, we used to call Pandoc to convert .md to .tex and .pdf separately)
-    if (output_format$pandoc$keep_tex || knitr::is_latex_output()) {
+    if (output_format$pandoc$keep_tex || knitr::is_latex_output() || pandoc_to %in% c("latex", "beamer")) {
```

Comment:
```suggestion
    if (output_format$pandoc$keep_tex || pandoc_to %in% c("latex", "beamer")) {
```

Reply:
Ok great ! I was not sure about the `out_format("latex")` format part in `knitr::is_latex_output()`. I guess it is not that important for **rmarkdown**. 

Thanks!